assets/js/datastore.coffee

```diff
@@ -22,6 +22,10 @@ Currently, DataStore has a synchronous API.  This may need to change eventually.
       @_collapsedKey_ = (row) -> "#{@prefix}:#{row}:collapsed"
       @_marksKey_ = (row) -> "#{@prefix}:#{row}:marks"
 
+      @_pluginRowKey_ = (plugin, row, key) -> "#{@prefix}:#{plugin}:#{row}:#{key}"
```

Comment:
`#{@prefix}:#{row}:plugin:#{plugin}:#{key}`, for consistency and avoiding conflict, perhaps?